Firestore/core/src/firebase/firestore/util/executor_libdispatch.mm

```diff
@@ -272,9 +272,10 @@ void MarkDone() {
                    TimeSlot::InvokedByLibdispatch);
 
   return DelayedOperation{[this, time_slot_id] {
-    // `time_slot` might be destroyed by the time cancellation function runs.
-    // Therefore, don't access any methods on `time_slot`, only use it as
-    // a handle to remove from `schedule_`.
+    // `time_slot` might have been destroyed by the time cancellation function
+    // runs, in which case it's guaranteed to have been removed from the
+    // schedule_. If the `time_slot_id` refers to a slot that has been removed,
```

Comment:
Ultranit: please add backticks around `schedule_`, for consistency with the rest of the comment.

Reply:
Done.